spec/requests/version_close_spec.rb

```diff
@@ -4,25 +4,41 @@
 
 RSpec.describe 'Close a version', type: :request do
   let(:druid) { FactoryBot.build(:workflow_step).druid }
-  let(:object_client) { instance_double(Dor::Services::Client::Object, version: version_client) }
-  let(:version_client) { instance_double(Dor::Services::Client::ObjectVersion, current: '2') }
 
   before do
-    allow(Dor::Services::Client).to receive(:object).with(druid).and_return(object_client)
     allow(QueueService).to receive(:enqueue)
   end
 
-  context 'deprecated route' do
+  context 'when version is not passed (deprecated)' do
+    let(:object_client) { instance_double(Dor::Services::Client::Object, version: version_client) }
+    let(:version_client) { instance_double(Dor::Services::Client::ObjectVersion, current: '2') }
+
+    before do
+      allow(Dor::Services::Client).to receive(:object).with(druid).and_return(object_client)
+    end
+
+    context 'with repository route (deprecated)' do
```

Comment:
should there be one of these clauses below?  asking for a friend.

Reply:
I'm not following what you are asking for.

Reply:
at line 38, shouldn't there be an outer clause stating it's without the repository route?